app/views/tag/blog.html.erb

```diff
@@ -1,36 +1,35 @@
 <div class="col-md-2">
-
 </div>
 <div class="col-md-8 blog">
   <div class="blog-header">
-    <h1><%= params[:id] == "blog" ? "The Public Lab" : params[:id].split('-').join(' ').capitalize %> Blog</h1>
+    <h1><%= params[:id] == "blog" ? "The Public Lab" : params[:id].split('-').join(' ').capitalize %> <%= t('tag.blog.blog') %></h1>
```

Comment:
This is an odd one, because we say `The Public Lab Blog` or `Foo Blog` based on whether it's a topical blog. But I think the string "The Public Lab" ought to be translated too, thanks!